core/src/input/input.tsx

```diff
@@ -50,14 +46,19 @@ export interface InputProps {
 	setValue?: (value: string) => void;
 
 	/**
-	 * Id of a [datalist][1] element to be used. Can pass an object with values for
-	 * the Input component to create the [datalist][1].
+	 * Id of a [datalist][3] element to be used. Can pass an object with values for
+	 * the Input component to create the [datalist][3].
 	 *
-	 * [1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist
+	 * [3]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist
```

Comment:
This is weird.. the change from "1" to "3". I think you would not need this since the numbering reset on every comment block